### PR TITLE
Add freelancing guide

### PR DESCRIPTION
**Pull request by StarfallProjects**
Per discussion in Slack, working on adding: https://techwritertoolkit.com/consultant/introduction/

### CODE REVIEW

File: docs/hiring-guide/index.rst [hunk 1]
```diff
@@ -32,104 +32,35 @@ If you are willing to share your story with the community, you can contact jobs@
...
-
-Interview questions better avoided (and alternatives)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-* Very in-depth questions about the subject matter, not about the writing skills.
-* Who are you as a person and what's your passion? -> Can you tell me about a project you worked on that you are proud of?
-* What's your five year plan? -> What's the next thing you want to learn?
-* What's your favorite food? (or other arbitrary ice-breakers) -> Anything more relevant to the job/workplace! Perhaps there's a social group in the office/common hobby to mention?
-* What would be your superpower? -> Would you rather be able to fly or be invisible? (to elicit their reasoning)
+Interview questions
```
Comment (plaindocs) [inline docs/hiring-guide/index.rst:32]: I'd add something like 

```
.. contents:: Table of Contents
    :depth: 3
```

around line 15 (and delete the tired note while you're there ;-)

File: docs/hiring-guide/index.rst [hunk 2]
```diff
@@ -32,104 +32,35 @@ If you are willing to share your story with the community, you can contact jobs@
...
+Consulting and freelancing
+--------------------------
 
-* Git or GUI for Git (such as SourceTree)
-* DITA and Markdown
-* Madcap Flare, Adobe Technical Communications Suite, or AuthorIt
-* HTML and CSS
-* Command line skills. CLI is very useful for working with Git (can be easier than GUI), when writing Linux or Unix-related documentation, and as a common language with developers and QA engineers.
+.. toctree::
+   :maxdepth: 2
```
Comment (plaindocs) [inline docs/hiring-guide/index.rst]:
```suggestion
   :maxdepth: 1
```

File: docs/hiring-guide/interview-questions.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
+Interview questions
```
Comment (plaindocs) [inline docs/hiring-guide/interview-questions.rst:1]: Nice, moving this into it's own page :+1:

**Review by plaindocs — COMMENTED** [review]
This is great! Super useful stuff.

If we fix up the links I'm happy to merge this in. 

I'd be tempted to refactor the intro somehow, maybe breaking it out into the Hiring page or something, cos it's a dissapointing click otherwise (IMO)

Are you planning to maintain the templates? :smiling_imp: :

**Comment by StarfallProjects** [comment]
@plaindocs re. the templates - I think it's nice to provide them in something like google docs, that people can easily copy and edit. Is there a WTD google drive (or similar)? If not, they're sat in an old account of mine, and I'm happy to leave them there. But yeah would be nice to have them somewhere more accessible.

**Review by plaindocs — APPROVED** [review]
Looks good to me! 
Any volunteers from @ericholscher @mxsasha @thatdocslady for an optional second set of eyes?

**Pull request merged**